pysat/instruments/pysat_testing.py

```diff
@@ -177,6 +187,8 @@ def load(fnames, tag='', inst_id='', sim_multi_file_right=False,
 
     if tag == 'default_meta':
         return data, pysat.Meta()
+    elif tag == 'no_download':
+        return pds.DataFrame(), pysat.Meta()
```

Comment:
```suggestion
    # TODO(#1120): Move logic up so that empty data is returned first.
    if tag == 'default_meta':
        return data, pysat.Meta()
    elif tag == 'no_download':
        return pds.DataFrame(), pysat.Meta()
```